statement.go

```diff
@@ -242,6 +240,13 @@ func (s *Stmt) bindJSON(val driver.NamedValue, n int) (mapping.State, error) {
 	return mapping.StateError, addIndexToError(unsupportedTypeError("JSON interface, need []byte or string"), n+1)
 }
 
+func (s *Stmt) bindUUID(val driver.NamedValue, t Type, n int) (mapping.State, error) {
```

Comment:
Do we already have a test for binding UUIDs? If not, could you add one for this code path? 

And a small nit: we do not need to pass `t Type` here.